cmd/run_test.go

```diff
@@ -147,11 +147,11 @@ func Test_runCmdWithMultiTenancy(t *testing.T) {
 		runCmd.Print(output)
 		// Check if GatewayD started and stopped correctly.
 		assert.Contains(t, output, "GatewayD is running")
-		assert.Contains(t, output, "There are clients available in the pool count=10 name=default")
-		assert.Contains(t, output, "There are clients available in the pool count=10 name=test")
+		assert.Contains(t, output, "There are clients available in the pool count=10 group=default name=default")
+		assert.Contains(t, output, "There are clients available in the pool count=10 group=test name=test")
```

Comment:
The line generating this log should be changed and the [`name`](https://github.com/gatewayd-io/gatewayd/blob/890d5620fdae947196390c4cab4b35871e767c83/cmd/run.go#L689) should be removed, since it is redundant.